app/knights/knight.py

```diff
@@ -0,0 +1,54 @@
+from __future__ import annotations
+
+
+class Knight:
+    def __init__(
+        self,
+        name: str,
+        power: int,
+        hp: int,
+        armour: list[dict],
+        weapon: dict,
+        potion: dict,
+    ) -> None:
+        self.name = name
+        self.power = power
+        self.hp = hp
+        self.armour = armour
+        self.weapon = weapon
+        self.potion = potion

```

Comment:
also u can call method gear up here, so it will be no need to call it later - it will be called every time the new instance of knight is creating